src/client/article.ts

```diff
@@ -370,10 +373,30 @@ function hydrateQuizAtoms(): void {
 		const props = atom.querySelector('.js-quiz-params')?.innerHTML;
 		try {
 			if (props) {
-				const quizProps: unknown = JSON.parse(
+				const quizProps = (JSON.parse(
 					props.replace(/&quot;/g, '"'),
-				);
-				ReactDOM.hydrate(h(QuizAtom, quizProps as QuizAtomType), atom);
+				) as unknown) as { quizType: string };
```

Comment:
Longer term we should probably parse this, rather than casting it, to avoid runtime errors. For now atoms aren't available to users, and this gets us closer to Emotion 11, so I think it's ok.

Reply:
Agreed 👍